src/Telemetry/AzureDevOpsTimeline/ITimelineTelemetryRepository.cs

```diff
@@ -0,0 +1,16 @@
+﻿using Microsoft.DotNet.Internal.AzureDevOps;
+using System;
+using System.Collections.Generic;
+using System.Threading.Tasks;
+
+namespace Microsoft.DotNet.AzureDevOpsTimeline
+{
+    public interface ITimelineTelemetryRepository : IDisposable
+    {
+        public Task<DateTimeOffset?> GetLatestTimelineBuild(string project);
```

Comment:
the "public" in interfaces is a little weird. :-) But maybe that's just my old fogie hat

Reply:
Probably just my fingers' muscle memory.

Reply:
Yeah. I don't think the style guidelines have been updated to account for "interface methods can be non-public now".

Reply:
I'm going to leave this unless anyone feels strongly about it. I have no problem being explicit.